cosmos/operators/kubernetes.py

```diff
@@ -59,16 +59,14 @@ def build_kube_args(self, context: Context, cmd_flags: list[str] | None = None)
         # to add that in the future
         self.dbt_executable_path = "dbt"
         dbt_cmd, env_vars = self.build_cmd(context=context, cmd_flags=cmd_flags)
-        
+
         # Parse ProfileConfig and add additional arguments to the dbt_cmd
-        if self.profile_config.profiles_yml_filepath:
-            with self.profile_config.ensure_profile() as (profile_path, _):
-                dbt_cmd.extend(["--profiles-dir", str(profile_path.parent)])
```

Comment:
Since `profile_yml_filepath` is tested against the local airflow `profiles.yml` file's location, and the docker image's environment most likely will differ, including this argument makes little sense.